infra_event_notifier/backends/slack.py

```diff
@@ -1,3 +1,51 @@
-def send_notification(title: str, text: str, slack_api_key: str) -> None:
-    # TODO: implement
-    pass
+import hashlib
+import hmac
+import json
+import urllib.request
+from urllib.error import HTTPError
+
+
+def send_notification(
+    title: str,
+    text: str,
+    channel_id: str,
+    eng_pipes_key: str,
+    eng_pipes_url: str,
+) -> None:
+    """
+    Sends an event to Slack via eng-pipes.
+    See: https://github.com/getsentry/eng-pipes
+
+    :param title: Title of Slack Message
+    :param text: Body of Slack Message
+    :param channel_id: ID of the Slack channel to send the message to
+    :param eng_pipes_key: Secret Key used to HMAC sign request
+    :param eng_pipes_url: Full URL for eng-pipes slack webhooks
+    """
+    # API docs: https://docs.datadoghq.com/api/latest/events/#post-an-event
+    payload = {
+        "source": "kafka-control-plane",
+        "title": title,
+        "body": text,
+        "channel": channel_id,
```

Comment:
`channel_id` was removed from the eng-pipes webhook, we just use a single channel hardcoded in eng-pipes for now